client/src/page-not-found/fallback-link.tsx

```diff
@@ -2,10 +2,14 @@ import React from "react";
 import { useLocation, useParams } from "react-router-dom";
 import useSWR from "swr";
 
-import { Doc } from "../../../libs/types/document";
-import LANGUAGES_RAW from "../../../libs/languages";
+import { Doc, LanguageItem } from "../../../libs/types";
 import NoteCard from "../ui/molecules/notecards";
 
+const LANGUAGES_RAW = require("../../../libs/languages") as Record<
+  string,
+  LanguageItem
+>;
```

Comment:
Could we move this (and the same in other files) into something like `libs/languages/index.ts` so we can `import` in typescript, while continuing to `require()` in js?

Reply:
Fixed in 7e75de5e4.

PS: This is possible because of: https://github.com/mdn/yari/pull/7006